components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1901,24 +1894,12 @@ void RewardsServiceImpl::SavePublisherInfo(
       return DeferCallback(FROM_HERE, std::move(callback),
```

Comment:
Similar to above, I think we want to always execute this line if `!Connected()`.

Reply:
Fixed in e63215cfbb11475a2d950311715154ff937ca143